- fix 'EXTERNAL_FRONTEND_HOST' webclient config option problem
- use specific rails version in the webclient
Index: ../1.0/src/webclient/config/environment.rb
===================================================================
--- ../1.0/src/webclient/config/environment.rb	(revision 4388)
+++ ../1.0/src/webclient/config/environment.rb	(working copy)
@@ -3,6 +3,7 @@
 # Uncomment below to force Rails into production mode when
 # you don't control web/app server and can't set it the proper way
 # ENV['RAILS_ENV'] ||= 'production'
+RAILS_GEM_VERSION = "1.2.6"
 
 # Bootstrap the Rails environment, frameworks, and default configuration
 require File.join(File.dirname(__FILE__), 'boot')
